extensions/db-diagram/src/ui-resources/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
```

Comment:
It would be good to go over the design as to why you're using a webview. If we can avoid using one that'd be best - is there a specific need for this or are you just testing things out?